packages/js/README.md

```diff
@@ -41,14 +41,16 @@ Before you're able to use this Matomo Tracker you need to initialize Matomo with
 const MatomoInstance = new window.MatomoTracker({
   urlBase: 'https://LINK.TO.DOMAIN',
   siteId: 3, // optional, default value: `1`
-  userId: 'UID76903202', // optional, default value: `undefined`.
```

Comment:
This would introduce a breaking change, so I think for now we should leave it. The way we initialise the instance isn't perfect though, so we're going to think about how we can improve this.

Reply:
Since the version is v0 I would expect breaking changes in the API, but it can be reverted, no problem.

Reply:
Yeah, we will probably change the API soon so this can be done more explicitly with chained calls on the instance rather than passing all the possible options in a config. We don't want to put more breaking changes before that as it might confuse users.

Reply:
Do you mean that the new API will be something like this?
```js
instance
    .urlBase('https://LINK.TO.DOMAIN')
    .siteId(3)
    .setUserId('example_id')
    .disableCookies()
    .setSecureCookie(true)
    .setRequestMethod('POST');
```

Reply:
This would be awesome, too!

Reply:
Yeah, that is basically what I have in mind as well. That and some better JSDoc for the functions plus a generic function to push entries into `window._paq` in case we don't have the method in our library.

Reply:
I think that's really nice! Created an issue: https://github.com/Amsterdam/matomo-tracker/issues/176